libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -551,6 +551,14 @@ singleton AP_Mount method set_rate_target void uint8_t 0 UINT8_MAX float'skip_ch
 singleton AP_Mount method set_roi_target void uint8_t 0 UINT8_MAX Location
 singleton AP_Mount method get_attitude_euler boolean uint8_t 0 UINT8_MAX float'Null float'Null float'Null
 
+include AP_Winch/AP_Winch.h
+singleton AP_Winch rename winch
```

Comment:
```suggestion
include AP_Winch/AP_Winch.h depends APM_BUILD_COPTER_OR_HELI
singleton AP_Winch rename winch depends APM_BUILD_COPTER_OR_HEL
```

Reply:
Oppps, wrong syntax, I think this should work:
```suggestion
include AP_Winch/AP_Winch.h depends APM_BUILD_COPTER_OR_HELI
singleton AP_Winch depends APM_BUILD_COPTER_OR_HELI
singleton AP_Winch rename winch
```

Reply:
Ah, nevermind, I've found what works. txs!